.github/workflows/sanity.yml

```diff
@@ -19,3 +19,13 @@ jobs:
 
     - name: run misspell
       run: ./bin/misspell -error .
+
+  encoding:
```

Comment:
Since we are using the free plan, which has a limit of 20 concurrent jobs, I would rather create a step instead of creating a new job

Reply:
I think this has to be solved by upgrading the plan.
I've raised the ask to the TC and our representative early this week, we should see the change next week.

Reply:
ohhh i see! :)

Reply:
Yea. We should be moving from 20 -> 60 parallel jobs soon, if not already.